llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -4478,7 +4484,9 @@ Constant *SPIRVToLLVM::transInitializer(SPIRVValue *const spvValue, Type *const
     // in memory.
     assert(initializer->getType()->isIntOrIntVectorTy(1));
     assert(type->isIntOrIntVectorTy(32));
-    initializer = ConstantExpr::getZExt(initializer, type);
+    initializer = ConstantFoldCastOperand(Instruction::ZExt, initializer, type, m_m->getDataLayout());
+    if (!initializer)
+      return nullptr;
```

Comment:
Same as above - and even if returning nullptr is appropriate - it'll do that anyway since the next statement outside the if is a return of initializer.

Reply:
Changed to an assert.